### PR TITLE
feat(server): Weave cache time and additional features.

### PR DESCRIPTION
**Pull request by vanpelt**
Boom

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -282,7 +299,6 @@ resource "kubernetes_secret" "secret" {
   }
 
   data = {
-    "MYSQL"       = var.database_connection_string
-    "OIDC_SECRET" = var.oidc_secret
```
Comment (yogeshg) [inline main.tf:286]: before this PR, looks like these secrets were not explicitly set in the environment, if that's the case, how did mysql work? Was something just broken?

Comment (yogeshg) [reply main.tf:286]: ah, we gotta remove the static secrets OIDC_SECRET and MYSQL from lines 97 and 127. 
https://github.com/wandb/terraform-kubernetes-wandb/pull/35/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR97
and

**Review by yogeshg — APPROVED** [review]

File: main.tf [hunk 2]
```diff
@@ -118,12 +113,15 @@ resource "kubernetes_deployment" "wandb" {
...
           }
 
-          env {
-            name = "OIDC_SECRET"
-            value_from {
-              secret_key_ref {
-                name = local.app_name
-                key  = "OIDC_SECRET"
+          dynamic "env" {
+            for_each = var.other_wandb_secrets
```
Comment (vanpelt) [inline main.tf:117]: This is wrong, why did you remove the hard coded static secrets.  If you're gonna do it this way this needs to be `local.all_secrets`...

File: main.tf [hunk 3]
```diff
@@ -3,6 +3,11 @@ locals {
   weave_app_name     = "weave"
   parquet_app_name   = "parquet"
   redis_ca_cert_name = "server_ca.pem"
+  static_secrets     = {
+    "MYSQL"       = var.database_connection_string
+    "OIDC_SECRET" = var.oidc_secret
+  }
+  all_secrets        = merge(local.static_secrets, var.other_wandb_secrets)
```
Comment (vanpelt) [inline main.tf:10]: This is the important line

File: main.tf (hunk 2 above)
Comment (yogeshg) [reply main.tf:137]: gah, sorry for the confusion, reverted the change from your branch. I kinda prefer using local.all_secrets, so here's stacked PR: https://github.com/wandb/terraform-kubernetes-wandb/pull/36

File: main.tf (hunk 1 above)
Comment (vanpelt) [reply main.tf:286]: Hmm, did that work?  I think you merged that PR into this one but it's still showing the wrong change against master.

**Review by yogeshg — APPROVED** [review]
Looking good!

**Comment by jsbroks** [comment]
This PR is included in version 1.11.0 :tada:
